todo.py

```diff
@@ -3,18 +3,44 @@ def print_todos(todos):
     if len(todos) == 0:
         print("You have nothing to do.")
     else:
-        i = 0
-        for todo in todos:
-            print(f"{i}: {todo}")
-            i += 1
+        # i = 0
+        # for todo in todos:
+        #     print(f"{i}: {todo}")
+        #     i += 1
+        print("====Pending=======")
+        for key in todos.keys():
+            if todos[key]['completed'] == False:
+                print(f"{todos[key]['index']}. {todos[key]['title']}")
+        print("==================")
+        print("====Completed=====")
+        for key in todos.keys():
+            if todos[key]['completed'] == True:
+                print(f"{todos[key]['index']}. {todos[key]['title']}")
+        print("==================")
+
+
+            # ========Pending========
+            # 0. titles in dictionary
+            # 1. title in dictionary
+            # =======================
+            # ========Completed======
+            # index number. title
+            # =======================
 
 def add_todo(todos, item):
-    todos.append(item)
+    # todos.append(item)
+    todos[item] = {
+        "title": item,
+        "completed": False,
+        "index": len(todos)
```

Comment:
Creative way to generate an index :)
It seems like you're trying to get your dictionary to simulate some of the behavior of a list...